mediacapture-image/ImageCapture-track.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>ImageCapture track</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/mediacapture-image/#dom-imagecapture-track">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<canvas id='canvas' width=10 height=10></canvas>
+<script>
+
+test(t => {
+  let canvas = document.getElementById('canvas');
+  let context = canvas.getContext('2d');
+  context.fillStyle = 'red';
+  context.fillRect(0, 0, 10, 10);
+
+  let stream = canvas.captureStream();
+  let videoTrack = stream.getVideoTracks()[0];
+  assert_equals(videoTrack.readyState, "live");
+  assert_true(videoTrack.enabled);
+
+  let capturer = new ImageCapture(videoTrack);
+  assert_true(capturer.track instanceof MediaStreamTrack);
+  assert_equals(capturer.track, videoTrack);
+
+  capturer.track.enabled = false;
```

Comment:
Is this really the proper definition of `readonly`. The `MediaStreamTrack` returned by the attribute getter should be the same (and just as modifiable through mutable attributes and exposed methods) as `videoTrack` is above.

Reply:
Yes, you are right, my mistake. Test the `readonly` attribute with a clone `MediaStreamTrack` now, PTAL.